test/sql/join/iejoin/iejoin_issue_10122.test

```diff
@@ -0,0 +1,40 @@
+# name: test/sql/join/iejoin/iejoin_issue_10122.test
+# description: Issue #10122: wrong result in IEJoin
+# group: [iejoin]
+
+statement ok
+pragma enable_verification
+
+statement ok
+create table tt (x int, y int, z int);
+
+statement ok
+insert into tt select nullif(r % 10, 0), nullif (r % 11, 0), r from range(50) tbl(r);
```

Comment:
Can we reduce the count to 15 and list the values explicitly? The hash makes it hard to see that the result is correct.